README.md

```diff
@@ -93,7 +93,7 @@ Click any of the links to serve the respective app:
 
 ### Terminal configutation
 
-You can also add a terminal(s) to the configuration file, in a similar way.
+You can also add a terminal to the configuration file, in a similar way.
```

Comment:
My suggested change assumes that immediately below we explain how exactly to add multiple terminals.
Other possible wordings:

```suggestion
You can also add one or more terminals to the configuration file, in a similar way.
```


```suggestion
You can also add terminals to the configuration file, in a similar way.
```